replay-protected-sighash.md

```diff
@@ -46,9 +46,15 @@ The proposed digest algorithm computes the double SHA256 of the serialization of
 
 Items 1, 4, 7 and 9 have the same meaning as in the original algorithm[[3]](#OP_CHECKSIG).
 
+#### outpoint
+
+An outpoint, as used in this document is two fields.
+1. a hash (txid) of the output's containing transaction. (32-byte hash)
+2. the ouput-index (4-byte little endian)
+
 #### hashPrevouts
 
-* If the `ANYONECANPAY` flag is not set, `hashPrevouts` is the double SHA256 of the serialization of all input outpoints;
+* If the `SIGHASH_ANYONECANPAY` flag is not set, `hashPrevouts` is the double SHA256 of the serialization of all input outpoints;
```

Comment:
I'm not convinced that putting SIGHASH all over improve anything. We do it is C because scoping doesn't work right there.